js/tabs.js

```diff
@@ -9,17 +9,23 @@ bcpl.tabs = (($) => {
 		const $targetTabControl = $(event.currentTarget);
 		const $tabs = event.data.$tabContainer.find(tabSelector);
 		const tabControlIndex = $targetTabControl.index();
+		const $activatedTab = $tabs.eq(tabControlIndex);
 
 		event.data.$tabControls.removeClass('active');
```

Comment:
you should be able to add these together wtih [.add](https://api.jquery.com/add/)